ptx/sec_deriv_prodquot.ptx

```diff
@@ -711,1446 +697,1395 @@
+        </exercise>
 
-      <exercise>
-        <webwork seed="1">
-            <setup>
+        <exercise>
+          <webwork seed="1">
             <pg-code>
-              $f = 'f';
-              $x = 't';
-              Context()->variables->are($x => 'Real');
-              $formula = Formula("$x^5(sec($x) + e^$x)");
-              $deriv = Formula("5$x^4*(sec($x)+e^$x)+t^5*(sec($x)*tan($x)+e^$x)");#$formula->D("$x")->reduce;
+              $fname = list_random('f','g','h','j','k','p');
+              Context()->variables->are(theta=>['Real',TeX=>'\theta']);
+              $x = Formula('theta');
+              $n = random(2,6,1);
+              $trig = list_random('cos','sin','sec','csc','tan','cot');
+              if($envir{problemSeed}==1){$fname='f';$n=3;$trig='sin';};
+              $formula = Formula("theta^$n $trig(theta) + $trig(theta)/theta^$n");
+              $df = $formula->D("$x");
             </pg-code>
-            </setup>
             <statement>
               <p>
-                <m><var name="$f"/>(<var name="$x"/>) = <var name="$formula"/></m>
+                <m><var name="$fname"/>(<var name="$x"/>) = <var name="$formula"/></m>
               </p>
-
+              <instruction>
+                To enter <m>\theta</m>, type <c>theta</c>.
+              </instruction>
               <p>
-                <m><var name="$f"/>'(<var name="$x"/>) = </m><var name="$deriv" width="15"/>
+                <var name="$df" width="80"/>
               </p>
             </statement>
-        </webwork>
-      </exercise>
+          </webwork>
+        </exercise>
 
-      <exercise>
-        <webwork seed="1">
-            <setup>
+        <exercise>
+          <webwork seed="1">
             <pg-code>
-              $f = 'f';
-              $x = 'x';
+              $fname = list_random('f','g','h','j','k','p');
+              $x = list_random('q','r','t','x','y','z');
+              ($triga,$trigb) = ('cos','sin','sec','csc','tan','cot')[NchooseK(6,2)];
+              if($envir{problemSeed}==1){$fname='f';$x='x';$triga='cos';$trigb='tan';};
               Context()->variables->are($x => 'Real');
-              $formula = Formula("sin($x)/(cos($x)+3)");
-              $deriv = Formula("(1+3*cos($x))/(cos($x)+3)^2");#$formula->D("$x")->reduce;
+              $formula = Formula("$triga($x)/$x + $x/$trigb($x)");
+              $df = $formula->D("$x");
             </pg-code>
-            </setup>
             <statement>
               <p>
-                <m><var name="$f"/>(<var name="$x"/>) = <var name="$formula"/></m>
+                <m><var name="$fname"/>(<var name="$x"/>) = <var name="$formula"/></m>
               </p>
-
               <p>
-                <m><var name="$f"/>'(<var name="$x"/>) = </m><var name="$deriv" width="15"/>
+                <var name="$df" width="80"/>
               </p>
             </statement>
-        </webwork>
-      </exercise>
+          </webwork>
+        </exercise>
 
-    <exercise>
```

Comment:
Did you remove these two exercises, or add ww code and move them? (Sorry, hard to tell in the diff.)
Where there are ones like this without ww code, it's most likely these are new questions that Greg added in APEX V4 (after the mbx conversion). I put them in the right place, but didn't have time to code them as ww problems.

Reply:
I think this one is the WW problem at the new line 1351.